demo/data/flags.json

```diff
@@ -1,26 +1,45 @@
-[
-    {
+{
+    "beta": [{
```

Comment:
I guess the thinking was that we need to be able to turn a flag both on and off in both beta and live environments. This is what we thought would be the easiest way of doing it with minimal changes required to the existing code.